tests/docker/Dockerfile

```diff
@@ -162,6 +166,7 @@ COPY --chown=0:0 tests/docker/ssh /root/.ssh
 COPY --chown=0:0 tests/setup.py /root/tests/
 RUN python3 -m pip install --upgrade --force pip && \
     python3 -m pip install --force --no-cache-dir -e /root/tests/
+# RUN python3 -m pip install --no-cache-dir --no-binary confluent-kafka confluent-kafka==1.7.0 --use-pep517 -v
```

Comment:
Maybe this should go in a separate commit?

Reply:
Removed, it wasn't as needed as I thought it was.